src/utils.c

```diff
@@ -59,3 +63,33 @@ void convert_to_hex_str(char* destination,
     }
     destination[i * 2] = '\0';
 }
+
+#if defined(TARGET_STAX)
+
+static void message_rejection(void) {
+    send_response(0, false, false);
+    nbgl_useCaseStatus("Message\nrejected", false, ui_idle);
```

Comment:
Same remark in this file

Reply:
`nbgl_useCaseStatus` is form the SDK, I would have liked a full `snake_case` but it's not up to me